app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/CategorizedPicture.kt

```diff
@@ -9,7 +9,7 @@ import android.widget.ImageView
  */
 
 interface CategorizedPicture : Parcelable {
-    val id: String
+    val id: Any
```

Comment:
Sidenote: I'm honestly not against ditching Any for String given the number of casts we already have, but when was that changed?

Reply:
I don't know why this was a String so I changed it back but I have in my personal notes that I wanted to bring up changing the id to String during the next sprint meeting.